src/site/content/en/blog/css-clipping/index.md

```diff
@@ -0,0 +1,223 @@
+---
+title: How to use CSS clipping
+description: |
+  Using clipping in CSS can help us move away from everything in our designs looking like a box.
+  By using various basic shapes, or an SVG, you can create a clip path.
+  Then cut away the parts of an element you don't want to show.
+authors:
+  - rachelandrew
+date: 2020-09-02
+hero: hero.jpg
+alt: Scissors hanging on a wall.
+tags:
+  - blog
+  - css
+---
+
+Elements on web pages are all defined inside a rectangular box.
+However that doesn't mean that we have to make everything look like a box.
+You can use the CSS `clip-path` property to clip away parts of an image or other element,
+to create interesting effects.
+
+<div class="glitch-embed-wrap" style="height: 420px; width: 100%;">
+  <iframe
+    src="https://glitch.com/embed/#!/embed/clip-path?path=README.md&previewSize=100"
+    title="clip-path on Glitch"
+    allow="encrypted-media"
+    style="height: 100%; width: 100%; border: 0;">
+  </iframe>
+</div>
+
+In the example above, the balloon image is square.
+Using `clip-path` and the basic shape value of `circle()`
+the additional sky around the balloon is clipped away leaving a circular image on the page.
+
+As the image is a link you can see something else about the `clip-path` property.
+Only the visible area of the image can be clicked on,
+as events do not fire on the hidden parts of the image.
+
+Clipping can be applied to any HTML element, not just images.
+There are a few different ways to create a `clip-path`, in this post we will take a look at them.
+
+## Browser compatibility
+
+Other than the box values as explained later in the post,
+the various values of `clip-path` demonstrated have [excellent browser support](https://caniuse.com/#feat=css-clip-path).
+For legacy browsers a fallback may be to allow the browser to ignore the `clip-path` property and show the unclipped image.
+If this is a problem you could test for `clip-path` in a feature query and offer an alternate layout for unsupporting browsers.
```

Comment:
Just realizing something… is `@supports` itself supported in legacy browsers? If it's IE11 then I suppose not… https://caniuse.com/mdn-css_at-rules_supports_selector

Reply:
the way you tend to use feature queries however is to do your old layout, then overwrite it in the query so it doesn't matter that IE11 neither supports the feature or the feature query as it ignores the whole lot.